start/sw/mobile-app/mobile-installation.md

```diff
@@ -8,4 +8,24 @@ description: Creating mobile applications using Google Android Studio.
 
 # Mobile application
 
-Today wide range of smart mobile devices make it possible to create clever applications. See [Google Android Studio](/start/sw/mobile-app/mobile-devassistant.html) for a step-by-step instructions to install Android Studio and create your own ```hello world``` mobile application.
+Today wide range of smart mobile devices make it possible to create clever applications.
+
+## Get started
+
+### Install required libraries
+
+```$ sudo dnf install glibc.i686 glibc-devel.i686 libstdc++.i686 zlib-devel.i686 ncurses-devel.i686```
```

Comment:
To clarify it- am I safe to remove all '.i686' suffixes from this command?

(I can edit it after merge if you do not want to edit your PR.)